rediscluster/client.py

```diff
@@ -1104,7 +1140,7 @@ def _random_good_hashslot_key(self, hashslot):
         Generate a good random key with a low probability of collision between any other key.
         """
         # TODO: Check if the key exists or not. continue to randomize until a empty key is found
-        random_id = "{{0}}{1}".format(hashslot, self._random_id())
+        random_id = "{{{0}}}{1}".format(hashslot, self._random_id())
```

Comment:
```python
def _random_good_hashslot_key(self, hashslot):
    """
    Generate a good random key with a low probability of collision between any other key.
    """
    # TODO: Check if the key exists or not. continue to randomize until a empty key is found
    random_id = "{{{0}}}{1}".format(hashslot, self._random_id())
    return random_id
```

You were passing two things to ``format`` but the format string only took one, as `{{0}}` is escaped. The version I've committed will produce:
```
'{hashslot}self._random_id'
```
Whereas the previous one *always* hashtagged into tag `0`:
```
'{0}self._random_id'
```
Ignoring the `hashslot` argument